src/containers/homepage/homepage.container.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Layout } from 'antd';
+import { SurveyListComponent } from './survey-list.component';
+
+const { Header, Footer, Sider, Content } = Layout;
+
+export class HomepageContainer extends Component {
+    render() {
+        const data = {
+            surveys: [
+                {
+                    name: 'Example Survey',
+                    createdAt: 0,
+                    lastEditedAt: 20,
+                    questionCount: 5,
+                    key: '0',
```

Comment:
This key is needed for React to distinguish items in an array (used in `antd/table` further on), is there any way it could auto populate this, or do we just provide something innately unique (like some hash of the - in this case - `localstorage` entry)

Reply:
Hash is unique but not stable, wdyt about using https://www.npmjs.com/package/uuid for this?

Reply:
I have used this package in commit `sha 236861d`